### PR TITLE
[chore] 0.3.1 release

### PR DESCRIPTION
**Pull request by min-xu-ai**
- mainly because vissl needs the new version
- added a doc on release steps

# Before submitting

- [ ] Was this discussed/approved via a Github issue? (no need for typos, doc improvements)
- [ ] Did you read the [contributor guideline](https://github.com/facebookresearch/fairscale/blob/master/CONTRIBUTING.md)?
- [ ] Did you make sure to update the docs?
- [ ] Did you write any new necessary tests?

## What does this PR do?
Fixes # (issue).

## PR review
Anyone in the community is free to review the PR once the tests have passed.
If we didn't discuss your PR in Github issues there's a high chance it will not be merged.

## Did you have fun?
Make sure you had fun coding 🙃

### CODE REVIEW

File: RELEASE.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+## Steps to do a release
```
Comment (blefaudeux) [inline RELEASE.md:1]: nice, thanks for this !

File: CHANGELOG.md [hunk 2]
```diff
@@ -6,8 +6,20 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
...
+## [0.3.1] - 2021-03-09
+### Added
+- FSDP docs ([#455](https://github.com/facebookresearch/fairscale/issues/455))
+- enable\_wrap and auto\_wrap APIs ([#446](https://github.com/facebookresearch/fairscale/issues/446))
+- experimental offload module ([#432](https://github.com/facebookresearch/fairscale/issues/432))
+
 ### Fixed
-- ShardedDDP auto catch trailing buckets (TBD)
+- Many FSDP fixes
+- Several SDP fixes around performance and corner cases
```
Comment (min-xu-ai) [inline CHANGELOG.md:21]: feel free to add more details here by doing GitHub "suggesting". Then, it is able to keep the co-author info.

File: CHANGELOG.md [hunk 3]
```diff
@@ -6,8 +6,20 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
...
+- AdaScale & SDP/FSDP test added but not officially supported
```
Comment (blefaudeux) [inline CHANGELOG.md:22]: would you mind adding:
- OSS: fix a broken state dict when using non contiguous param groups

**Review by blefaudeux — APPROVED** [review]
LGTM, thanks Min !

File: RELEASE.md (hunk 1 above)
Comment (min-xu-ai) [reply RELEASE.md:1]: I am easy to forget, so I often document everything, I mean EVERYTHING! lol

File: CHANGELOG.md [hunk 4]
```diff
@@ -6,8 +6,20 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
...
 
+### Added
+
+### Fixed
+
+## [0.3.1] - 2021-03-09
+### Added
+- FSDP docs ([#455](https://github.com/facebookresearch/fairscale/issues/455))
+- enable\_wrap and auto\_wrap APIs ([#446](https://github.com/facebookresearch/fairscale/issues/446))
+- experimental offload module ([#432](https://github.com/facebookresearch/fairscale/issues/432))
```
Comment (anj-s) [inline CHANGELOG.md:17]:
```suggestion
- Added experimental.nn.OffloadModel API for training large models on a single GPU.([#432](https://github.com/facebookresearch/fairscale/issues/432))
```

File: RELEASE.md (hunk 1 above)
Comment (anj-s) [reply RELEASE.md:1]: +1 on adding a doc for releases! Thank you @min-xu-ai

**Review by anj-s — APPROVED** [review]

**Pull request merged**